packages/ui/src/utils/PresetUtil.ts

```diff
@@ -18,21 +26,23 @@ export const PresetUtil = {
       tokenPocket: 'f3119826-4ef5-4d31-4789-d4ae5c18e400',
       tokenary: '5e481041-dc3c-4a81-373a-76bbde91b800',
       '1inch': 'dce1ee99-403f-44a9-9f94-20de30616500',
-      phantom: '62471a22-33cb-4e65-5b54-c3d9ea24b900'
+      phantom: '62471a22-33cb-4e65-5b54-c3d9ea24b900',
```

Comment:
Minor: Might have to be renamed phantomETH / phantomSOL wallet in near future?

Reply:
Yeh, I had similar thought actually. I think that presets will be moved out of `core` package into specific chain package instead, so id's for multichain wallets can stay the same, they'll just come from diff chain packages.